code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -153,12 +156,18 @@
 
 	return data
 
+#define CHEMMASTER_DISPENSE_SOUND playsound(src, 'sound/machines/reagent_dispense.ogg', 75, 1)
+#define CHEMMASTER_CHANGESETTINGS_SOUND playsound(src, 'sound/machines/slide_change.ogg', 75, 1)
+#define CHEMMASTER_SWITCH_SOUND playsound(src, 'sound/machines/switch1.ogg', 75, 1)
```

Comment:
The defines should be placed at the top unless there is a good reason not to do so.